src/main/java/org/team199/robot2020/commands/Shoot.java

```diff
@@ -0,0 +1,26 @@
+package org.team199.robot2020.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import org.team199.robot2020.subsystems.Shooter;
+import org.team199.robot2020.commands.EjectCell;
+
+public class Shoot extends CommandBase {
```

Comment:
the two commands `Shoot` and `EjectCell` are confusingly named. At least add comments to their constructors!